rma_repair/models/rma_order_line.py

```diff
@@ -75,8 +74,7 @@ def _get_rma_repaired_qty(self):
         qty = 0.0
         for repair in self.repair_ids.filtered(
                 lambda p: p.state != 'cancel'):
-            repair_qty = self.env['product.uom']._compute_qty_obj(
-                self.uom_id,
+            repair_qty = self.env['product.uom']._compute_quantity(
```

Comment:
in latests versions you have to call `_compute_quantity` from the uom you want to convert from. See https://github.com/odoo/odoo/blob/11.0/addons/product/models/product_uom.py#L92. So in this case it would be:
```python
repair_qty = self.uom_id._compute_quantity(
    repair.product_qty,
    repair.product_uom,
 )
```